app/assets/javascripts/components/util/ScrollToTop.jsx

```diff
@@ -0,0 +1,12 @@
+import { useEffect } from 'react';
+
+const ScrollToTop = () => {
```

Comment:
This one only scrolls to top when it mounts, right? As opposed to the `ScrollToTop` example from the docs that scrolls any time the route changes? If so, using the same naming as the docs probably makes sense for clarity.